core/arxiv/submission/domain/submission.py

```diff
@@ -70,7 +70,7 @@ class SubmissionContent:
 
 
 @dataclass
-class SubmissionCompiled:
+class SubmissionProduct:
```

Comment:
Indeed, I'm not sure what we need this class for, given the introduction of  ``Compilation``